bokun.yaml

```diff
@@ -5141,118 +5142,179 @@ definitions:
         format: double
   ActivityAvailabilityDto:
     properties:
+      id:
+        type: string
+        description: 'Value "startTimeId_YYYYMMDD" (YYYYMMDD - date of the departure for this particular availability).'
       activityId:
         type: integer
         format: int64
+        description: 'Id of the activity'
+      activityTitle:
+        type: string
+        description: 'Title of the activity.'
       activityOwnerId:
         type: integer
         format: int64
+        description: 'Id of vendor which owns activity (original supplier)'
+      activityOwnerTitle:
+        type: string
+        description: 'Title of vendor which owns activity (original supplier)'
+      startTime:
+        type: string
+        description: 'Start time in hours:minutes using 24h format (e.g. "19:00"). If this is not a time based availability, e.g. pass type, then value is "00:00"'
+      startTimeId:
+        type: integer
+        format: int64
+        description: 'Unique system id for the start time mapped to this availability'
+      startTimeLabel:
+        type: string
+        description: 'Optional label if used for this start time'
+      flexible:
+        type: boolean
+        description: 'If activity "bookingType" is DATE then true, otherwise - false.'
+      date:
+        type: integer
+        format: int64
+        description: 'Timestamp for request current date (at time exactly 0:00 UTC).'
```

Comment:
I'd rather spell this as `Date of a particular availability in the time zone of relevant experience product`

Reply:
The idea using term "timestamp" was to specify that value is in such format: "date": 1585094400000 , as in other properties of this response there are used different formats for date.
And time zone remark is correct, thanks!